### DISCUSSION TITLE
Credentials request stuck in pending state

### DISCUSSION THREAD

**Discussion by lindeberg** [nextauthjs/next-auth#3029] [Help] [answered]
![image](https://user-images.githubusercontent.com/15940340/138837532-15b5173f-77ca-441f-a086-9344febff4a3.png)
![image](https://user-images.githubusercontent.com/15940340/138837538-0b7280f7-d942-4149-b1a6-a25f7ed78c90.png)
![image](https://user-images.githubusercontent.com/15940340/138837546-771d82c3-e8b1-45d5-a636-2618315332d0.png)
![image](https://user-images.githubusercontent.com/15940340/138837622-c0a208bf-8e17-4068-9d66-a56f9b9d1b02.png)

After calling the signIn-method with credentials provider to custom backend. I can see that the backend receives and responds.

**Comment by balazsorban44** [comment, marked as answer]
have a repro?

**Reply by lindeberg** [reply]
No

**Reply by balazsorban44** [reply]
could you please create one so we can help?

**Reply by lindeberg** [reply]
I figured it out! Was returning an object instead of null when the signing wasn't successful.
